packages/faceted-search/stories/facetedSearch.stories.js

```diff
@@ -523,15 +523,46 @@ export const WithQuickSearchFilter = () => (
 	</FacetedSearch.Faceted>
 );
 
-export const WithQuickSearchFilterCustomizableInputTriggerLength = () => (
-	<FacetedSearch.Faceted id="my-faceted-search">
-		<p>Quick search will trigger after a minimum input length that can be customized</p>
-		<br />
-		<FacetedSearch.BasicSearch
-			badgesDefinitions={badgesDefinitions}
-			callbacks={callbacks}
-			onSubmit={action('onSubmit')}
-			quickSearchMinLength={5}
-		/>
-	</FacetedSearch.Faceted>
-);
+export const WithQuickSearchFilterCustomizableInputTriggerLength = () => {
+	const quickSearchMinLength = 5;
+	const badgeNameWithLength = {
+		properties: {
+			attribute: 'name',
+			initialOperatorOpened: true,
+			initialValueOpened: false,
+			label: 'Name',
+			operator: {},
+			operators: [],
+			type: 'text',
+			placeholder: 'Enter a dataset name',
+		},
+		metadata: {
+			isAvailableForQuickSearch: true,
+			isAvailableForFacetList: true,
+			badgePerFacet: 'N',
+			entitiesPerBadge: '1',
+			operators: [
+				'containsIgnoreCase',
+				'notContainsIgnoreCase',
+				'equals',
+				'notEquals',
+				'match a regexp',
+			],
+			'data-feature': 'faceted-badge-name',
+			minLength: quickSearchMinLength,
```

Comment:
if I understand it well you have to repeat the quickSearchMinLength twice.
Could it be taken from the props ?

Reply:
Yes you are right i just changed everything because it could be a mess to configure things everywhere